src/options/options.css

```diff
@@ -1,81 +1,9 @@
-body {
-  /* the style the other options in the settings pane use */
-  font-size: 1.11em;
-  color: #333;
-}
-
-/* on (small) mobile displays */
-body.mobile {
-  /* smaller size -> default on Firefox for Android is 14px */
-  font-size: 14px;
-}
-
-/* disable all options incomaptible with mobile devices */
-body.mobile .mobile-incompatible {
-  display: none;
-}
-
-/* https://design.firefox.com/photon/patterns/inactive.html */
-[disabled] {
-  opacity: 0.4;
-}
-
-ul {
-  margin: 0px;
-  padding: 0px;
-}
-li {
-  list-style-type: none;
-  margin-top: 10px;
-  padding: 0px;
-}
-/* in a fieldset e.g. we use a condensed version */
-.condensed-list {
-  margin-left: 0px;
-}
-
-fieldset {
-  /* same border style as separators above */
-  border: 1px solid;
-  border-radius: 4px;
-  border-color: var(--grey-30); /* Firefox: --in-content-box-border-color */
-
-  /* a big width, but not too big */
-  display: inline-block;
-  width: auto;
-  min-width: 60vw;
-
-  margin-bottom: 4px;
-}
-
-/* when a setting has a label *before* it, add some margin */
-label + .setting {
-  margin-left: 8px;
-}
-
-.helper-text {
-  display: block;
-  color: var(--grey-50);
-
-  margin-top: 4px;
-}
-
-/* some margin to align with checkbox */
-input[type=checkbox] ~ .helper-text,
-input[type=radio] ~ .helper-text {
-  /* 4px margin-left + 16px size + 3px margin-right + 5px space (#text) */
-  margin-left: 28px;
-}
+@import "/common/modules/AutomaticSettings/css/photonOptions.css";
 
-/* when a link is used in a helper text, add margin */
-.helper-text > a {
-  margin-left: 4px;
-  overflow-wrap: none;
+.line > code {
+  margin: 0;
 }
 
-/* workaround for missing invalid style, see https://bugzilla.mozilla.org/show_bug.cgi?id=1514567 */
-.browser-style > input[type=text]:invalid:not(:focus) {
-  border-color: var(--red-60); /* makes border quare for some reason on GNOME */
-  border-radius: 2px; /* looks better on GNOME, at least */
-  box-shadow: 0 0 0 2px var(--red-60-light-shadow);
+.float-image {
```

Comment:
BTW why did you leave that in here? No image is included yet.
That said, you can leave it here, I’ll include more images later. :smiley:

Reply:
I just copied this file verbatim from the Awesome Emoji Picker after I realized that the options page did not look right.